client/components/seo-preview-pane/index.jsx

```diff
@@ -0,0 +1,100 @@
+/** @ssr-ready **/
+
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import PureComponent from 'react-pure-render/component';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import SearchPreview from 'components/seo/search-preview';
+import VerticalMenu from 'components/vertical-menu';
+import { SocialItem } from 'components/vertical-menu/items';
+import { getSelectedSite } from 'state/ui/selectors';
+
+const ComingSoonMessage = translate => (
+	<div className="seo-preview-pane__message">
+		{ translate( 'Coming Soon!' ) }
+	</div>
+);
+
+const GooglePreview = site =>
+	<SearchPreview
+		title={ site.name }
```

Comment:
Thanks @roundhill - it's not a finished product, but this PR and PRs built on top of it are getting large and hard to manage. I was hoping to push this out as a stepping-stone to the final feature.